src/modules/units/components/UnitGroupCard.tsx

```diff
@@ -76,5 +71,21 @@ const UnitGroupCard: React.FC<Props> = ({
       )}
     </Paper>
   );
+
+  if (linkToUnitGroup) {
+    return (
+      <CardActionArea
+        component={RouterLink}
+        to={generateSafePath(ProjectDashboardRoutes.UNIT_GROUP, {
+          projectId,
+          unitGroupId: unitGroup.id,
+        })}
+        aria-label='Manage Unit Group'
```

Comment:
```suggestion
        aria-label={`Manage Unit Group ${unitGroup.name}`}
```
also something to note, all of this card's interior content is hidden to screen readers in favor of the aria label, including the "x/y units vacant".

Reply:
Thanks, added. I think it's probably OK for that content to be hidden since it's shown elsewhere on the page. 

I'm facing issues with <a> tag being nested inside <a> tag because of the settings button inside the clickable card. I think I'm going to remove the button for now since the accessibility of the clickable card appears OK to me. Let me know if you have a different suggestion though, we can come back to it